node/malus/src/malus.rs

```diff
@@ -36,14 +36,6 @@ enum NemesisVariant {
 	BackGarbageCandidate(BackGarbageCandidateOptions),
 	/// Delayed disputing of ancestors that are perfectly fine.
 	DisputeAncestor(DisputeAncestorOptions),
-
-	#[allow(missing_docs)]
-	#[command(name = "prepare-worker", hide = true)]
-	PvfPrepareWorker(polkadot_cli::ValidationWorkerCommand),
```

Comment:
Do we have tests for malus? I tried running `0001-dispute-valid-block.toml` but I got `error: unexpected argument '--unsafe-ws-external' found` - not sure where that argument is even coming from since it's not in the toml file.

Reply:
OK I think that's why the zombienet job was failing. Fixed, thanks for the catch. ⭐️